### PR TITLE
What about #8019

### PR DESCRIPTION
**Pull request by ChillerDragon**
This is not a pullrequest. This is an issue.

https://github.com/ddnet/ddnet/pull/8019 just introduced a strict style guide. And it has been already violated. This should either be checked manually before new prs get merged. Or fixed on a regular basis. Or better added to ./scripts/fix_style.py

### CODE REVIEW

File: src/game/client/components/menus_settings.cpp [hunk 1]
```diff
@@ -1164,7 +1164,7 @@ float CMenus::RenderSettingsControlsJoystick(CUIRect View)
 
 			View.HSplitTop(Spacing, nullptr, &View);
 			View.HSplitTop(ButtonHeight, &Button, &View);
-			Ui()->DoScrollbarOption(&g_Config.m_UiControllerSens, &g_Config.m_UiControllerSens, &Button, Localize("UI controller sens."), 1, 500, &CUi::ms_LogarithmicScrollbarScale, CUi::SCROLLBAR_OPTION_NOCLAMPVALUE);
+			Ui()->DoScrollbarOption(&g_Config.m_UiControllerSens, &g_Config.m_UiControllerSens, &Button, Localize("Ui controller sens."), 1, 500, &CUi::ms_LogarithmicScrollbarScale, CUi::SCROLLBAR_OPTION_NOCLAMPVALUE);
```
Comment (def-) [inline src/game/client/components/menus_settings.cpp:1167]: This is wrong, it's not an identifier

File: src/game/client/components/menus_settings.cpp [hunk 2]
```diff
@@ -1331,7 +1331,7 @@ void CMenus::RenderSettingsControls(CUIRect MainView)
 			MouseSettings.HSplitTop(2.0f, nullptr, &MouseSettings);
 
 			MouseSettings.HSplitTop(20.0f, &Button, &MouseSettings);
-			Ui()->DoScrollbarOption(&g_Config.m_UiMousesens, &g_Config.m_UiMousesens, &Button, Localize("UI mouse sens."), 1, 500, &CUi::ms_LogarithmicScrollbarScale, CUi::SCROLLBAR_OPTION_NOCLAMPVALUE);
+			Ui()->DoScrollbarOption(&g_Config.m_UiMousesens, &g_Config.m_UiMousesens, &Button, Localize("Ui mouse sens."), 1, 500, &CUi::ms_LogarithmicScrollbarScale, CUi::SCROLLBAR_OPTION_NOCLAMPVALUE);
```
Comment (def-) [inline src/game/client/components/menus_settings.cpp:1334]: wrong

File: src/game/client/components/menus_settings.cpp [hunk 3]
```diff
@@ -1689,7 +1689,7 @@ void CMenus::RenderSettingsGraphics(CUIRect MainView)
 
 	MainView.HSplitTop(2.0f, nullptr, &MainView);
 	static CButtonContainer s_UiColorResetId;
-	DoLine_ColorPicker(&s_UiColorResetId, 25.0f, 13.0f, 2.0f, &MainView, Localize("UI Color"), &g_Config.m_UiColor, color_cast<ColorRGBA>(ColorHSLA(0xE4A046AFU, true)), false, nullptr, true);
+	DoLine_ColorPicker(&s_UiColorResetId, 25.0f, 13.0f, 2.0f, &MainView, Localize("Ui Color"), &g_Config.m_UiColor, color_cast<ColorRGBA>(ColorHSLA(0xE4A046AFU, true)), false, nullptr, true);
```
Comment (def-) [inline src/game/client/components/menus_settings.cpp:1692]: wrong

File: src/game/client/ui.cpp [hunk 4]
```diff
@@ -28,7 +28,7 @@ void CUIElement::Init(CUi *pUI, int RequestedRectCount)
 
 void CUIElement::InitRects(int RequestedRectCount)
 {
-	dbg_assert(m_vUIRects.empty(), "UI rects can only be initialized once, create another ui element instead.");
+	dbg_assert(m_vUIRects.empty(), "Ui rects can only be initialized once, create another ui element instead.");
```
Comment (def-) [inline src/game/client/ui.cpp:31]: wrong

**Review by def- — CHANGES_REQUESTED** [review]

File: src/game/client/lineinput.h [hunk 5]
```diff
@@ -15,7 +15,7 @@
 enum class EInputPriority
 {
 	NONE = 0,
-	UI,
+	Ui,
```
Comment (archimede67) [inline src/game/client/lineinput.h:18]: Wrong, this is an enum

**Comment by ChillerDragon** [comment]
xd guys I am aware that it is wrong. But then the fix script needs to be improved and integrated into the CI. I just ran the command @heinrich5991 built.

This is not a pullrequest that should be merged. It just shows that running this has a diff which it shouldn't:

```
sed -i \
        -e 's/\([a-z]_\?\)ID/\1Id/g' \
        -e 's/\([^ ]\)\<UI\>/\1Ui/g' \
        -e 's/UI()/Ui()/g' \
        -e 's/\<CUI\>/CUi/g' \
        -e 's/\([\ta-z.(&]\|[,=|] \)ID\>/\1Id/g' \
        -e 's/\<ID\>\([^ ").]\)/Id\1/g' \
        -e 's/\<ID\([0-9]\)/Id\1/g' \
        -e 's/\<ID\>\( [<=>:+*/-]\)/Id\1/g' \
        -e 's/int ID/int Id/g' \
        -e 's/\([a-z]_\?\)GPU/\1Gpu/g' \
        -e 's/\([a-z]_\?\)IP/\1Ip/g' \
        -e 's/\([a-z]_\?\)CID/\1Cid/g' \
        -e 's/\([a-z]_\?\)MySQL/\1Mysql/g' \
        -e 's/MySql/Mysql/g' \
        -e 's/\([a-xz]_\?\)SQL/\1Sql/g' \
        -e 's/DPMode/DpMode/g' \
        -e 's/TTWGraphics/TTwGraphics/g' \
        \
        -e 's/Ipointer/IPointer/g' \
        -e 's/\.vendorId/.vendorID/g' \
        -e 's/\.windowId/.windowID/g' \
        -e 's/SDL_GetWindowFromId/SDL_GetWindowFromID/g' \
        -e 's/SDL_AudioDeviceId/SDL_AudioDeviceID/g' \
        -e 's/SDL_JoystickId/SDL_JoystickID/g' \
        -e 's/SDL_JoystickInstanceId/SDL_JoystickInstanceID/g' \
        -e 's/AVCodecId/AVCodecID/g' \
        src/**/*.cpp src/**/*.h {datasrc,scripts}/**/*.py
git checkout -- src/engine/external
```

Not all of them are false positives and #8019 is not very long ago. If we keep introducing style violations the huge reformatting commit was not really worth it. And if the fix script has false positives it can not be used by branches or forks to automatically resolve conflicts.

File: src/game/client/components/menus_settings.cpp [hunk 6]
```diff
@@ -1832,15 +1832,15 @@ void CMenus::RenderSettingsGraphics(CUIRect MainView)
...
+				s_vpGpuIdNames[i] = aCurDeviceName;
 				if(str_comp("auto", g_Config.m_GfxGpuName) == 0)
 				{
 					OldSelectedGpu = 0;
 				}
 			}
 			else
 			{
-				s_vpGpuIDNames[i] = GPUList.m_vGpus[i - 1].m_aName;
+				s_vpGpuIdNames[i] = GPUList.m_vGpus[i - 1].m_aName;
```
Comment (Robyt3) [inline src/game/client/components/menus_settings.cpp:1843]: The script should also change `GPU` -> `Gpu` (except in strings).

**Comment by ChillerDragon** [comment]
I wonder if it would be smarter to use something like tree sitter here instead of a naive script.

**Comment by Robyt3** [comment]
If you want to fix/check variable names in the CI for all our conventions then I don't think naive scripts will work.

1. Use C++ parser to get all variable declarations + locations
2. Filter based on location (ignore external etc.)
3. Filter based on variable name (ignore i and other allowed short variable names)

**Comment by ChillerDragon** [comment]
I just wanted to show that heinrichs script causes a diff which is not nice. It is not solved yet but I don't want this weird non pr to become stale. So closing this.

**Pull request closed without merging**